### DISCUSSION TITLE
关于hg38.refseq.txt文件问题

### DISCUSSION THREAD

**Discussion by ZHONGHONGXIN** [ShixiangWang/self-study#115] [Q&A] [answered]
老师，您好；请问您知道如何下载hg38.refseq.txt这个文件？ 或者如何从hg38.refGene.txt转换为hg38.refseq.txt？谢谢您。

**Comment by ShixiangWang** [comment]
我不理解这两种文件有什么区别

**Reply by ZHONGHONGXIN** [reply]
老师，我遇到的情况是：在使用cworld-dekker处理HiC-Pro结果（matrix2compartment.pl），由于比对使用的是hg38，所以导致报错：refseq file missing! [cworld-dekker/scripts/util/geneDensity/hg38.refseq.txt]。
但是我目前还没找到办法下载hg38.refseq.txt。

**Reply by ShixiangWang** [reply]
这种问题建议你仔细看下软件的使用文档。如果 hg38.refseq.txt 是一个不常用参考文件名词，是不是把 hg38.refGene.txt 重命名一下可以解决。

**Comment by ZHONGHONGXIN** [comment, marked as answer]
谢谢王老师哈，我在Github（https://github.com/gimelbrantlab/magic/find/master）找到这个文件，然后成功运行出结果啦；还是要谢谢您回复消息。
祝好！

**Reply by ShixiangWang** [reply]
enen
